addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -444,9 +445,6 @@
                 <xpath expr="//div[hasclass('oe_chatter')]" position="attributes">
                     <attribute name="class" remove="o-aside" add="o-full-width" separator=" "/>
                 </xpath>
-                <xpath expr="//notebook/page[@name='mail_body']//field[@name='body_arch']" position="attributes">
```

Comment:
Actually for stable you can keep it, I don't think it will create any clash, would it ? That way we lessen diff and avoid any potential issue.

Reply:
yes, we can keep it.